cmd/kubectl-gadget/undeploy.go

```diff
@@ -65,7 +70,13 @@ func init() {
 		&undeployWait,
 		"wait", "",
 		true,
-		"wait for all resources to be deleted before returning",
+		"wait for all Inspektor Gadget resources to be deleted before returning",
+	)
+	undeployCmd.PersistentFlags().BoolVarP(
+		&deleteNamespace,
+		"delete-namespace", "",
+		false,
+		"delete the entire namespace (WARNING: this will remove ALL resources in the namespace, not just Inspektor Gadget)",
```

Comment:
Rather than having the warning as a comment here, we should create a prompt asking users to confirm the deletion with `Y/n` with the possibility to disable it using using `--force` ?

Reply:
IMO its okay the way it is. Maybe even without the warning.

The flag is called `delete-namespace` and is disabled by default